app-nordigen/README.md

```diff
@@ -0,0 +1,61 @@
+# Integration new bank
+
+Find in [doc](https://docs.google.com/spreadsheets/d/1ogpzydzotOltbssrc3IQ8rhBLlIZbQgm5QCiiNJrkyA/edit#gid=489769432) what is id of bank which you want to integrate
+
+Add the `institution_id` and your name to list of possible options in the frontend 
+project `actual/packages/loot-design/src/components/modals/NordigenExternalMsg.js`
+
+```
+<Strong>Choose your banks:</Strong>
+<CustomSelect
+  options={[
+    ['default', 'Choose your bank'],
+    ['ING_PL_INGBPLPW', 'ING PL'],
+    ['MBANK_RETAIL_BREXPLPW', 'MBANK'],
+    ['SANDBOXFINANCE_SFIN0000', 'DEMO - TEST']
+  ]}
+```
```

Comment:
hm… would it be reasonable to offer an API on the server side that returns this list of valid banks for display in the UI? That way the frontend and backend can’t fall out of sync. The frontend doesn’t need any individual handling for these, right?

Reply:
That's correct, I put it in the description of the PR as one of the next steps to do. 
I have very limited time to work on this. I'm working on this PR since Aug 2022 ;) Each work on the FE side takes me too much time. I know that I will not have it soon. Maybe we can leave it as a next improvement.

Reply:
Sure!